README.md

```diff
@@ -57,29 +52,25 @@ Returns a readable stream.
 
 Type: `string` [Possible values](presets)
 
-A set of options of a popular project so you don't have to define everything in options, context, gitRawCommitsOpts, parserOpts or writerOpts manually. The preset values can be overwritten.
+It's recommended to use a preset so you don't have to define everything yourself.
+
+Defaults to 'standard'.
 
 ##### pkg
 
 Type: `object`
 
+An object representing package.json.
+
 ###### path
 
 Type: `string` Default: `'package.json'`
 
-The location of your "package.json".
```

Comment:
There are two options named `transform`. I deleted one, but I'm not really sure what's up there.